spec/spec_helper.rb

```diff
@@ -19,7 +19,6 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
   config.filter_run_when_matching :focus
-  config.filter_run_excluding algolia: true
```

Comment:
i thought this was a bit odd

Reply:
It was only because the integration tests made indexes and records and so should have been used sparingly.

Reply:
@alexbowen it was needed so that expensive specs with `algolia: true` weren't executed every time, but instead only executed if `rspec --tag algolia` was explicitly called